dockstore-webservice/src/main/java/io/swagger/api/impl/ToolsImplCommon.java

```diff
@@ -206,11 +222,17 @@ private static String extractHTTPPrefix(String gitUrl, String reference, String
             version.setId(tool.getId() + ":" + inputVersion.getName());
 
             version.setName(inputVersion.getName());
-
             version.setVerified(inputVersion.isVerified());
             version.setVerifiedSource(Strings.nullToEmpty(inputVersion.getVerifiedSource()));
             version.setDockerfile(false);
 
+            if (inputVersion instanceof Tag) {
+                Tag tag = (Tag)inputVersion;
+                version.setImage(tag.getImageId());
+            } else {
+                version.setImage("");
```

Comment:
Seems odd, seems like if the default was null, that would be preferred. If the default was "", then this is just redundant

Reply:
Several things to note:
- TRSV will fail any properties that are null (protobuf also doesn't like null)
- Null-value properties can be ignored during serialization global using `mapper.setSerializationInclusion(JsonInclude.Include.NON_NULL);`
- I currently don't know how to modify the mapper for a specific endpoint or set of endpoints